src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -0,0 +1,79 @@
+// libs
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+// platform - forms - selectors
+import { preSubmitSelector } from 'platform/forms/selectors/review';
+
+// platform - forms-system components
+import { PreSubmitSection as DefaultPreSubmitSection } from 'platform/forms-system/src/js/components/PreSubmitSection';
+
+// platform - form-system actions
+import { setPreSubmit as setPreSubmitAction } from 'platform/forms-system/src/js/actions';
+
+/*
+*  RenderPreSubmitSection - renders PreSubmitSection by default or presubmit.CustomComponent 
+*  PreSubmitSection - ~Default component that renders if no CustomComponent is provided~ (this describes a decision in RenderPreSubmitSection- describe what PreSubmitSection is, remove this since it's not a prop, or add it as a prop with a default value)
+*  preSubmitInfo.CustomComponent - property that can be added to `preSubmitInfo` object that overwrites `PreSubmitSection`
```

Comment:
I'm not sure I understand this. I see no `RenderPreSubmitSection` or `preSubmitInfo.CustomComponent`. (There is a `preSubmit.CustomComponent`, however. Should that be `preSubmitInfo`?)

Reply:
This is just copy pasta from moving the component around.
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/review/SubmitController.jsx#L80

Reply:
In that case, it's probably worth taking a closer look at this and massaging it into something that makes more sense given its new home.